app/js/utils/helpers.js

```diff
@@ -91,24 +91,23 @@ export const getResultValue = (data) => {
 };
 
 export const getConceptShortName = (concept, locale) => {
```

Comment:
@cioan Hope you don't mind -- I did a very slight refactoring of this function, just to use more compact syntax.